base-containers/base/bin/INGInious

```diff
@@ -196,11 +201,28 @@ class INGIniousMainRunner:
                                          "memory_limit": message["memory_limit"],
                                          "share_network": message["share_network"],
                                          "socket_id": message["socket_id"],
-                                         "ssh": message["ssh"]})
+                                         "ssh": message["ssh"],
+                                         "run_as_root": message["run_as_root"]})
+                return False
+            if message["type"] == "run_student_init":  # This message might be sent from run_student to transfer to the student_container via the agent
```

Comment:
"might"?

Reply:
"may" is maybe clearer. 
The idea is to note the fact this message will only be received if we start a student_container on kata runtime.
The comment is inproved right now.